### PR TITLE
Add libcrack

### PR DESCRIPTION
**Pull request by marijuanARM**

### CODE REVIEW

File: build_info/libcrack-dev.control [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Package: libcrack-dev
```
Comment (1Conan) [inline build_info/libcrack-dev.control:1]: missing depends

File: build_info/libcrack2.control [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Package: libcrack2
```
Comment (1Conan) [inline build_info/libcrack2.control:1]: missing depends

**Review by 1Conan — CHANGES_REQUESTED** [review]

File: build_info/libcrack-dev.control (hunk 1 above)
Comment (marijuanARM) [reply build_info/libcrack-dev.control:1]: fixed in 5690bdd

File: build_info/libcrack-bin.control [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Package: libcrack-bin
```
[the remark this answers is not in the document]
Comment (marijuanARM) [reply build_info/libcrack-bin.control]: fixed in 5690bdd

**Review by 1Conan — APPROVED** [review]

**Pull request merged**